### PR TITLE
Fixed the error

### PR DESCRIPTION
**Pull request by sashuu6**
I have fixed the following errors:-
1. Fixed intent error present at line 43 in app.py
2. Fixed import issue in enhance/image_enhance.py and enhance/ridge_freq.py
3. Fixed type error at line 88 and 90 in enhance/ridge_filter.py

### CODE REVIEW

**Comment by sashuu6** [comment]
I have deleted the __py_cache__ folder present in enhance module.

File: app.py [hunk 1]
```diff
@@ -86,13 +86,13 @@ def main():
 	img4 = cv2.drawKeypoints(img1, kp1, outImage=None)
 	img5 = cv2.drawKeypoints(img2, kp2, outImage=None)
 	f, axarr = plt.subplots(1,2)
-	axarr[0].imshow(img4)
-	axarr[1].imshow(img5)
-	plt.show()
+	# axarr[0].imshow(img4)
```
Comment (kjanko) [inline app.py]: ?

File: app.py [hunk 2]
```diff
@@ -86,13 +86,13 @@ def main():
...
+	# axarr[1].imshow(img5)
```
Comment (kjanko) [inline app.py]: ?

File: app.py [hunk 3]
```diff
@@ -86,13 +86,13 @@ def main():
...
+	# plt.show()
```
Comment (kjanko) [inline app.py]: ?

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "python-env/bin/python2.7"
```
Comment (kjanko) [inline .vscode/settings.json]: Remove this file.

**Comment by sashuu6** [comment]
I have commented the lines to automate the process.
I will fix it right now.

**Comment by sashuu6** [comment]
i have added all the requested code.

**Pull request merged**